randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -140,12 +144,19 @@ Extra - asset_id: 3
   * Normal Door to West Ocean/Door to Moat
   > Door to Crateria Kihunter Room
       Any of the following:
-          # Via top path - TODO: check for shinespark without entrance rando?
-          Grapple Beam or Space Jump
+          Any of the following:
+              # Via top path
+              Grapple Beam or Space Jump or Movement (Intermediate)
+              # Start in West Ocean and jump over with momentum
+              Speed Booster and Disabled Door Lock Randomizer and Disabled Entrance Randomizer
           Any of the following:
               # Via bottom path
               Can Lay Bombs
-              Gravity Suit and Screw Attack
+              All of the following:
+                  Morph Ball
+                  Any of the following:
```

Comment:
My lines 155-157 look different. I'm not sure what to do here. 
![image](https://github.com/randovania/randovania/assets/46821362/33461b80-15f8-415c-adbc-df41d1ecdc09)

Reply:
That's the spot i'm talking about, but you changed stuff with screw, so it now looks different. 

But my pojnt was, instead of making it like this:
![image](https://github.com/randovania/randovania/assets/38186597/e9f7b840-1c5c-47de-b76c-080eb3f2e662)

Make it like this for maintaibility
![image](https://github.com/randovania/randovania/assets/38186597/e334ce43-5744-490a-b580-52d2cbb66a19)


That way if there are other things that require morph, you can more easily add them that way under the OR box